### PR TITLE
add test coverage for closing filter popover

### PR DESCRIPTION
**Pull request by kimberlythegeek**

### CODE REVIEW

**Comment by kimberlythegeek** [comment]
so I haven't had a chance to look at some of the other uncovered lines, but I was able to write a test to add coverage for close the popover.

**Review by esanuandra — COMMENTED** [review]
This test is showing some errors/warnings in the console. Could you look into that?
` console.error
      Warning: An update to Transition inside a test was not wrapped in act(...).
      When testing, code that causes React state updates should be wrapped into act(...):
      act(() => {/* fire events that update state */
}); /* assert on the output */`

File: src/__tests__/CompareResults/CompareResultsTable.test.tsx [hunk 1]
```diff
@@ -260,4 +260,31 @@ describe('Compare Results Table', () => {
...
+      name: 'windows10-64-shippable-qr',
+    });
+    expect(windowsCheckbox).toBeInTheDocument();
+
+    await user.click(windowsCheckbox);
+
+    const applyButtons = screen.getAllByRole('button', { name: 'Apply' });
+
+    await user.click(applyButtons[0]);
+    jest.runOnlyPendingTimers();
```
[the remark this answers is not in the document]
Comment (kimberlythegeek) [reply src/__tests__/CompareResults/CompareResultsTable.test.tsx]: I'm not sure what all can cause this problem, I believe it's related to `useFakeTimers`, but without it the test fails. I wrapped this line in `act` and the warning no longer appears

**Review by esanuandra — APPROVED** [review]

**Comment by esanuandra** [comment]
Looks good. It needs an update and then it can be merged.

**Pull request merged**